icu4j/main/core/src/main/java/com/ibm/icu/message2/MessageFormatter.java

```diff
@@ -319,6 +367,26 @@ public Builder setPattern(String pattern) {
             return this;
         }
 
+        /**
+         * Sets the {@link ErrorHandlingBehavior} to use when encountering errors at formatting time.
```

Comment:
Can you refer back to the comment on `ErrorHandlingBehavior` so as to avoid duplicating this text?

Reply:
Removed the duplicated paragraph.

The first paragraph already contains a link to `ErrorHandlingBehavior` which contains more info.